azure-pipelines.yml

```
@@ -1,3 +1,8 @@
+trigger:
+  branches:
+    include:
+    - '*'  
+    - refs/tags/*
```

Since we only run the release task on Python tags (currently detected with `v1.*`) maybe we should only trigger additional builds on those tags as well?

```suggestion
    - refs/tags/v1.*
```